spec/plugin.spec.js

```diff
@@ -85,7 +88,8 @@ describe('ManifestPlugin', function() {
       }, {}, function(manifest) {
         expect(manifest).toBeDefined();
         expect(manifest).toEqual({
-          'main.js': 'main.js'
+          'main.js': 'main.js',
+          'third.party.js': 'third.party.js'
```

Comment:
One of the reason I tried to use first `webpackConfig` was to be able to run the specs with/without adding the mocking plugin. If you have an idea how could this be added (testing with and without using plugin), please give me some hints and I can added the code for it. Right now is always used in specs, i think it would be a nice to have in tests.